src/shared/lib/FormatUtils.tsx

```diff
@@ -61,8 +61,15 @@ export function toFixedWithThreshold(value: number, digits: number): string {
     return fixed;
 }
 
-export function getPercentage(proportion: number, digits: number = 1) {
-    return `${toFixedWithThreshold(100 * proportion, digits)}%`;
+export function getPercentage(proportion: number) {
```

Comment:
@jjgao confirmed we can show <1%, which means we don't have to sorry about decimals anymore.